src/main/main.ts

```diff
@@ -538,7 +538,12 @@ function initializeIpc() {
     }
   });
 
-  ipcMain.on("relaunch standalone", async (event) => {
+  ipcMain.on("relaunch standalone", async (event, param: {}) => {
```

Comment:
The type `{}`, confusingly, includes primitive types such as numbers and strings.

Personally I think it's better to use dedicated type like `Record<"reason", string>`, but either way looks okay to me.

```suggestion
  ipcMain.on("relaunch standalone", async (event, param: object) => {
```

Reply:
I intend for give choice to caller. I guess `object` is looks good to me.